tests/test_z_cmdline.py

```diff
@@ -8,6 +8,7 @@
 import pytest
 
 import tox
+import tox.exc
```

Comment:
not too happy with this ``exc``, can we make it something more descriptive?

Reply:
yep - we can have a module with the full name instead I think.